dashboard/db/migrate/20240320051410_add_issuer_to_lti_user_identities.rb

```diff
@@ -0,0 +1,7 @@
+class AddIssuerToLtiUserIdentities < ActiveRecord::Migration[6.1]
+  def change
+    add_column :lti_user_identities, :issuer, :string
+    change_column_null :lti_user_identities, :issuer, false
+    add_index :lti_user_identities, :issuer
```

Comment:
Just a demonstration of how it can be accomplished in a single line
```suggestion
    add_column :lti_user_identities, :issuer, :string, null: false, index: true
```

Reply:
Yeah, this is the first time I am doing migrations, at first, I was experimenting and just created the column.
Then I realized I could not be null and we needed to index it, but the column was already created in my local DB

Reply:
You can always rollback your migration: `cd dashboard && rails db:rollback`